HealthVault/Classes/Types/MHVThingQuery.h

```diff
@@ -44,6 +44,7 @@
 
 @property (readwrite, nonatomic) int maxResults;
 @property (readwrite, nonatomic) int maxFullResults;
+@property (readwrite, nonatomic) BOOL ignoreCache;
```

Comment:
maybe shouldUseCachedResults would be more clear. Also, we should explicitly set a default value when this object gets initialized.